snippets/capitalize-first-letter.md

```diff
@@ -1,10 +1,9 @@
 ### Capitalize first letter
 
-Use `slice(0,1)` and `toUpperCase()` to capitalize first letter, `slice(1)` to get the rest of the string.
+Use `Array.from()` and `toUpperCase()` to capitalize first letter.
+
 Omit the `lowerRest` parameter to keep the rest of the string intact, or set it to `true` to convert to lower case.
 
 ```js
-const capitalize = (str, lowerRest = false) =>
-  str.slice(0, 1).toUpperCase() + (lowerRest? str.slice(1).toLowerCase() : str.slice(1));
-// capitalize('myName', true) -> 'Myname'
-```
+const capitalize = (word, forceLowerCase = false) =>  Array.from(forceLowerCase ? word.toLowerCase() : word, (l,i) => i === 0 ? l.toUpperCase() : l).join("")
```

Comment:
Why change `lowerRest` to `forceLowerCase`? I think it was fine as it was.

Reply:
yeah, just felt it was more explicit to the action it performs, not that important!

Reply:
I'll run jsPerf in the morning and see if there's any major performance hit. Otherwise, this is solid. I still prefer `lowerRest`, but I might be too opinionated on that, so leave it any way you prefer it. Thanks for the help!